systers_portal/templates/meetup/add_meetup.html

```diff
@@ -39,9 +39,9 @@
   })
 
   $('.timepicker').pickatime({
-    format: 'HH:i',
-    formatSubmit: 'HH:i',
+    format: 'H:i',
+    formatLabel: 'h:ia',
```

Comment:
Ah, it's format label that changed. Ok. Why did you remove formatSubmit at all?
Also you can adjust the display of date and time in template to whichever format you want - https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#date